src/components/Group/Navbar.vue

```diff
@@ -1,38 +1,41 @@
 <template>
-    <q-tabs v-if="shouldBeDisplayed" align="center" color="tertiary" class="mobile-only shadow-14 inset-shadow" hide="icon" slot="navigation">
-        <q-route-tab
-            :to="{name: 'group'}"
-            exact
-            slot="title"
-            name="wall"
-        >{{ $t('GROUP.WALL') }}</q-route-tab>
-        <q-route-tab
-            :to="{name: 'groupDescription'}"
-            exact
-            slot="title"
-            name="description"
-        >{{ $t('GROUP.DESCRIPTION') }}</q-route-tab>
-        <q-route-tab
-            :to="{name: 'groupHistory'}"
-            exact
-            slot="title"
-            name="history"
-        >{{ $t('GROUP.HISTORY') }}</q-route-tab>
+  <div slot="navigation" class="shadow-14 inset-shadow row">
+    <q-tabs color="tertiary" hide="icon" class="col">
+      <q-route-tab
+        :to="{name: 'group'}"
+        exact
+        slot="title"
+        name="wall">
+        {{ $t('GROUP.WALL') }}
+      </q-route-tab>
+      <q-route-tab
+        :to="{name: 'groupDescription'}"
+        exact
+        slot="title"
+        name="description">
+        {{ $t('GROUP.DESCRIPTION') }}
+      </q-route-tab>
+      <q-route-tab
+        :to="{name: 'groupHistory'}"
+        exact
+        slot="title"
+        name="history">
+        {{ $t('GROUP.HISTORY') }}
+      </q-route-tab>
     </q-tabs>
+    <q-btn flat class="bg-tertiary text-white">
+      <q-icon name="fa-ellipsis-v" />
+      <GroupOptions/>
+    </q-btn>
+  </div>
 </template>
 
 <script>
-import { QTabs, QRouteTab } from 'quasar'
+import { QTabs, QRouteTab, QBtn, QIcon } from 'quasar'
+import GroupOptions from '@/components/Sidenav/GroupOptions'
 
 export default {
-  components: { QTabs, QRouteTab },
-  computed: {
-    shouldBeDisplayed () {
```

Comment:
Now that 'shouldBeDisplayed' is removed, the tabs are still around on the member page - and the last active one (history, description...) is still highlighted. I think we should either remove the tabs, or remove the highlighting

Reply:
Or probably we should just add the "members" option to the tabs. Don't think it's available somewhere else....

Reply:
I think it would be best if the last active one wouldn't be highlighted. Seems like a bug in quasar to me...
The check happens one component above, it checks if the route name starts with `group`. This is just a very basic workaround to have group navigation available on most group pages that are not in the top nav (members, invitations...)